scripts/Assignment_8_3.R

```diff
@@ -0,0 +1,59 @@
+# Assignment 1:  
+library(tweedie) 
+library(ggplot2)
+library(doParallel) #read doParallel function
+library(dplyr)
+library(tidyr)
+library(stringr)
+
+
+simTweedieTest <-  
+  function(N){ 
+    t.test( 
+      rtweedie(N, mu=10000, phi=100, power=1.9), 
+      mu=10000 
+    )$p.value 
+  } 
+
+#use more cores
+maxcores <- 8
+Cores <- min(parallel::detectCores(), maxcores)
+
+#Instantiate the cores:
+cl <- makeCluster(Cores)
+
+# Next we register the cluster..
+registerDoParallel(cl)
+
+
+#rewrite the replicate function, using parallel computing
+#I tried this, but unfortunately it didn't work...I don't know how to solve this
+MTweedieTests <-  
+  function(N,M,sig){ 
+    res <-
+      foreach(
+        i = 1:M,
+        .combine = 'rbind',
+        .packages = c('magrittr', 'dplyr')

```

Comment:
Function is almost correct.

You forgot to add the package "tweedie".
When you use functions that are stored in the GlobalEnvironment (functions you have created yourself) inside a function, you need to use the .export function. You should have included this argument:

```r
.export = c("simTweedieTest")
```